plugins/synaptics-mst/fu-synaptics-mst-firmware.c

```diff
@@ -6,16 +6,19 @@
 
 #include "config.h"
 
+#include "fu-synaptics-mst-common.h"
 #include "fu-synaptics-mst-firmware.h"
 
 struct _FuSynapticsMstFirmware {
 	FuFirmwareClass parent_instance;
 	guint16 board_id;
+	FuSynapticsMstFamily family;
 };
 
 G_DEFINE_TYPE(FuSynapticsMstFirmware, fu_synaptics_mst_firmware, FU_TYPE_FIRMWARE)
 
-#define ADDR_CUSTOMER_ID 0x10E
+#define ADDR_CUSTOMER_ID_CAYENNE 0X20E
+#define ADDR_CUSTOMER_ID 0X10E
```

Comment:
lower case `0x` please

Reply:
Fixed in [cdc7651](https://github.com/fwupd/fwupd/pull/6559/commits/cdc7651611c59e6fdc65c974ebb1c11734ce9480)